datasets/qa_org/test_data.sql

```diff
@@ -0,0 +1,2204 @@
+INSERT INTO "npe4__Relationship__c" VALUES(107,'Employee','90','72');
+INSERT INTO "npe4__Relationship__c" VALUES(108,'Employee','89','39');
+INSERT INTO "npe4__Relationship__c" VALUES(109,'Partner','89','88');
+INSERT INTO "npe4__Relationship__c" VALUES(110,'Employee','91','23');
+INSERT INTO "npe4__Relationship__c" VALUES(111,'Employee','63','33');
+INSERT INTO "npe4__Relationship__c" VALUES(112,'Employee','64','38');
+INSERT INTO "npe4__Relationship__c" VALUES(113,'Employee','65','86');
+INSERT INTO "npe4__Relationship__c" VALUES(114,'Employee','92','31');
+INSERT INTO "npe4__Relationship__c" VALUES(115,'Employee','93','5');
+INSERT INTO "npe4__Relationship__c" VALUES(116,'Employee','94','99');
+INSERT INTO "npe4__Relationship__c" VALUES(117,'Employee','95','96');
+INSERT INTO "npe4__Relationship__c" VALUES(118,'Employee','98','54');
+INSERT INTO "npe4__Relationship__c" VALUES(119,'Employee','99','94');
+INSERT INTO "npe4__Relationship__c" VALUES(120,'Employee','1','41');
+INSERT INTO "npe4__Relationship__c" VALUES(121,'Employee','100','70');
+INSERT INTO "npe4__Relationship__c" VALUES(122,'Employee','5','93');
+INSERT INTO "npe4__Relationship__c" VALUES(123,'Employee','2','21');
+INSERT INTO "npe4__Relationship__c" VALUES(124,'Employee','3','28');
+CREATE TABLE "CustomObject1__c" (
+    id INTEGER NOT NULL,
+    Name VARCHAR(255),
+    RecordTypeId VARCHAR(255),
+    Opportunity__c VARCHAR(255),
+    C1_rt_picklist__c VARCHAR(255),
+    C1_currency2__c VARCHAR(255),
+    C1_currency__c VARCHAR(255),
+    C1_date__c VARCHAR(255),
+    C1_number__c VARCHAR(255),
+    C1_phone__c VARCHAR(255),
+    C1_picklist__c VARCHAR(255),
+    C1_text2__c VARCHAR(255),
+    C1_text__c VARCHAR(255),
+    C1_textarea__c VARCHAR(255),
+    C1_url__c VARCHAR(255),
+    PRIMARY KEY (id)
+);
+INSERT INTO "CustomObject1__c" VALUES(1, 'Flower 1', 'Flower', '', 'Marigold', '', '', '', '', '', '', '', '', '', '');
+INSERT INTO "CustomObject1__c" VALUES(2, 'Flower 2', 'Flower', '', '', '', '', '', '', '', '', '', '', '', '');
+INSERT INTO "CustomObject1__c" VALUES(3, 'Vegetable 1', 'Vegetable', '', 'Potato', '', '', '', '', '', '', '', '', '', '');
+INSERT INTO "CustomObject1__c" VALUES(4, 'Vegetable 2', 'Vegetable', '', '', '', '', '', '', '', '', '', '', '', '');
+INSERT INTO "CustomObject1__c" VALUES(5, 'Fruit 1', 'Fruit', '', 'Apple', '', '', '', '', '', '', '', '', '', '');
+INSERT INTO "CustomObject1__c" VALUES(6, 'Fruit 2', 'Fruit', '', '', '', '', '', '', '', '', '', '', '', '');
+
+CREATE TABLE "CustomObject1__c_rt_mapping" (
+    record_type_id VARCHAR(255) NOT NULL,
+    developer_name VARCHAR(255),
+    PRIMARY KEY (record_type_id)
+);
+INSERT INTO "CustomObject1__c_rt_mapping" VALUES('Flower', 'Flower');
+INSERT INTO "CustomObject1__c_rt_mapping" VALUES('Vegetable', 'Vegetable');
+INSERT INTO "CustomObject1__c_rt_mapping" VALUES('Fruit', 'Fruit');
+
+-- fix for allowing namespaced scratch orgs to deploy these record types
```

Comment:
@psadie-sfdo Here's one possible solution for the namespaced scratch org problem with record types.

Reply:
I'd appreciate feedback from someone in RE on this one

Reply:
Could you share some additional context on the problem?

I would strongly encourage all teams to stop using namespaced scratch orgs. 2GP orgs are far superior for their intended purpose. There are some wrinkles for NPSP as 2GP orgs use regression configuration rather than QA, but that seems like a fruitful opportunity to unify those configurations.

Reply:
@davidmreed - The name of the record type table is the problem. In namespaced scratch orgs, the name of the CustomObject1__c object when you `Describe` it is `npsp__CustomObject1__c, and that value is what gets used to find the record type table - `npsp__CustomObject1__c_rt_mapping` which doesn't exist.

Reply:
That is a bug in CumulusCI. I will create a WI to fix it properly.